sqlmesh/core/model/definition.py

```diff
@@ -2771,9 +2781,15 @@ def render_field_value(value: t.Any) -> t.Any:
         field_value = fields.get(field)
 
         # We don't want to parse python model cron="@..." kwargs (e.g. @daily) into MacroVar
-        if field == "cron" or field_value is None:
+        if (
+            field == "cron" and isinstance(field_value, str) and field_value.lower() in CRON_ALIASES
+        ) or field_value is None:
             continue
 
+        # If it contains a macro reference we need to render it similar to sql models
+        if field == "cron" and isinstance(field_value, str):
+            field_value = exp.to_identifier(field_value)
```

Comment:
Is this needed? Won't the control flow reach L2820, where `field_value` will be parsed & rendered?